frappe/public/js/frappe/ui/capture.js

```diff
@@ -53,74 +74,229 @@ frappe.ui.Capture = class {
 		return this;
 	}
 
-	render() {
-		return navigator.mediaDevices.getUserMedia({ video: true }).then(stream => {
-			this.stream = stream;
+	show() {
+		this.build_dialog();
 
-			this.dialog = new frappe.ui.Dialog({
-				title: this.options.title,
-				animate: this.options.animate,
-				on_hide: () => this.stop_media_stream()
-			});
+		if (frappe.is_mobile()) {
+			this.show_for_mobile();
+		} else {
+			this.show_for_desktop();
+		}
+	}
 
-			this.dialog.get_close_btn().on('click', () => {
-				this.hide();
-			});
+	build_dialog() {
+		let me = this;
+		me.dialog = new frappe.ui.Dialog({
+			title: this.options.title,
+			animate: this.options.animate,
+			fields: [
+				{
+					fieldtype: "HTML",
+					fieldname: "capture"
+				},
+				{
+					fieldtype: "HTML",
+					fieldname: "total_count"
+				}
+			],
+			on_hide: this.stop_media_stream()
+		});
 
-			const set_take_photo_action = () => {
-				this.dialog.set_primary_action(__('Take Photo'), () => {
-					const data_url = frappe._.get_data_uri(video);
-					$e.find('.fc-p').attr('src', data_url);
+		me.$template = $(frappe.ui.Capture.TEMPLATE);
 
-					$e.find('.fc-s').hide();
-					$e.find('.fc-p').show();
+		let field = me.dialog.get_field("capture");
+		$(field.wrapper).html(me.$template);
 
-					this.dialog.set_secondary_action_label(__('Retake'));
-					this.dialog.get_secondary_btn().show();
+		me.dialog.get_close_btn().on('click', () => {
+			me.hide();
+		});
+	}
 
-					this.dialog.set_primary_action(__('Submit'), () => {
-						this.hide();
-						if (this.callback) this.callback(data_url);
-					});
-				});
-			};
+	show_for_mobile() {
+		let me = this;
+		if (!me.input) {
+			me.input = get_file_input();
+		}
 
-			set_take_photo_action();
+		me.input.onchange = async () => {
```

Comment:
The test is failing it says 
```SyntaxError: Can not use keyword 'await' outside an async function```

```suggestion
		me.input.onchange = async function() {
```
This will resolve the issue, but I don't why it is failing in v13 and not in develop branch.